pkg/secrets/clients/conjur/mocks/conjur_secrets_retriever.go

```diff
@@ -13,22 +13,44 @@ import (
 */
 var CanExecuteConjurVar bool
 
-func RetrieveConjurSecrets(_ []byte, variableIDs []string) (map[string][]byte, error) {
+type ConjurMockClient struct {
+	CanExecute bool
+	// TODO: CanExecute is really just used to assert on the presence of errors
+	// 	and should probably just be an optional error.
+	Database   map[string]string
+}
+
+func (c ConjurMockClient) RetrieveSecrets (_ []byte, variableIDs []string) (map[string][]byte, error) {
 	conjurSecrets := make(map[string][]byte)
 
-	if !CanExecuteConjurVar {
+	if !c.CanExecute {
 		return nil, errors.New("custom error")
```

Comment:
I know this error message is existing, but could we change the error message to something better?